src/opds_dumper.cpp

```diff
@@ -94,24 +95,24 @@ kainjow::mustache::object getSingleBookData(const Book& book)
     };
 }
 
-std::string getSingleBookEntryXML(const Book& book, bool withXMLHeader, const std::string& rootLocation, const std::string& endpointRoot, bool partial)
+std::string getSingleBookEntryXML(const Book& book, const NameMapper& nameMapper,  bool withXMLHeader, const std::string& rootLocation, const std::string& endpointRoot, bool partial)
```

Comment:
I started #829 in order to avoid ending up with such a function receiving 6 parameters.

Reply:
Yes, that's why I've started a factorization after this commit :)